data/language/en-GB.txt

```diff
@@ -4471,6 +4471,8 @@ STR_6161    :Gridlines display toggle
 STR_6162    :Spinning Wild Mouse
 STR_6163    :Mouse shaped cars speed through tight corners and short drops, gently spinning around to disorientate the riders
 STR_6164    :{WHITE}{CROSS}
+STR_6165    :Use Vsync
```

Comment:
This description is too technical IMO. How about `Synchronise to monitor refresh rate`?

Reply:
I don't agree, as Vsync is a very basic term in the game industry. Also, STR_6166 explains it further if people don't understand it. It really isn't an as andvanced, difficult and never-heard option as you think @Gymnasiast ......

Reply:
How about you stop sticking your nose into everything. You are not an average user. I don't care what "every game" does. Just stop being so bloody tedious every time.

Reply:
Uhhhhhh, I'm just giving my OPINION...... I'm not gonna react to this kind of messages. Oh, and if almost any game features this option (and also every page at PCGamingwiki talks about this) I think we can safely assume that this is an standard. I can of course be wrong...... @LRFLEW ?

Reply:
> Uhhhhhh, I'm just giving my OPINION

That's the whole bloody point. It's all you ever do. And it gets old fast.

Reply:
@Gymnasiast Honestly, I have to agree with @mrtnptrs here. VSync is a technical term, like "Nearest Neighbor" or "Linear". Your suggested name duplicates what's already in the hover text, and could cause confusion for people looking for the "VSync" option.

Side note: It seems that the spelling/capitalization of the term lacks consistency. Looking online, I've seen Vsync, VSync, and V-Sync. Even [Nvidia](https://www.geforce.com/hardware/technology/adaptive-vsync) doesn't seem to be completely consistent. We should probably decide on one of those three variants, or just stick with "Vertical Sync"

Reply:
`Use vertical synchronisation`, then? Clearer than VSync.
(Before you say it won't fit, remember that most languages do not shorten synchronisation to sync. Better use a full row.)

Reply:
Yeah, that should work. I really want to shorten it to `Use vertical sync`, though (not for length, but because I don't see synchronization spelled out often when talking about V-Sync).

Reply:
I fully agree with @LRFLEW here again.

@Gymnasiast Please don't be offended, but we all give our own opinions here, even you. What you say is also just an opinion..... I have as much right to give my opinion here as you. Even though you are a code contributor to OpenRCT2 and I'm not..... that doesn't give you the right to act like this when someone doesn't agree with you. So, shall we both shut up about this @Gymnasiast ? :) And by the way, I just tried to give some valuable feedback regarding G-sync in all this and I found a bug in this PR.

Reply:
Fine, make it `Use vertical sync`. But do make sure you reserve a full row for this, as this won't work in other languages.

Reply:
I swapped the "Show FPS" option back to where it was before, and gave the vertical sync option its own line. That should be enough space for the other languages.